config/config.auto.in

```diff
@@ -5,5 +5,8 @@ CL=@CC@
 CCFLAGS_NO_C=@CCFLAGS_NO_C@
 CCFLAGS=$(CCFLAGS_NO_C) -c
 CLFLAGS=-L. -L$(LIBDIR) @LDCCFLAGS@ -lm
-
+DEBUG_CFLAGS=@DEBUG_CFLAGS@
```

Comment:
@mreineck needs to comment on the inclusion of more autoconf macros like this. I suppose since it only changes the default configuration and there's plenty of opportunity for manual override it's not too dangerous/